content/20.treatments.md

```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
-
-In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
-The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
-However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
-These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
-The acute phase response to infection results in the heavy damage of the endothelium of blood vessels, which disrupts the balance between the pro- and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
-Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death in ARDS and Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus.
-Recently Chinese and Italian doctors have found that Tocilizumab (TCZ), or actemra by Roche, a drug commonly used to treat rheumatoid arthritis (RA), may palliate the most severe symptoms associated with COVID-19.
+TCZ is a receptor antibody that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine interleukin-6 (IL-6) [@doi:10.1101/cshperspect.a016295].
+IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, immune system regulators that are primarily responsible for immune cell differentiation.
+Often used to treat conditions such as rheumatoid arthritis [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19. 
+While secretion of IL-6 can be associated with chronic conditions, it is a key player in the innate immune response that is secreted by macrophages in response to the detection of PAMPs or DAMPs [@doi:10.1101/cshperspect.a016295].
+An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 were elevated at admission in adult COVID-19 patients who ultimately deceased compared to those who survived and remained higher, on average, throughout the course of hospitalization [@doi:10/ggnxb3].
```

Comment:
The second half of this sentence is a little confusing to me. So the IL-6 levels in those patients that were died were higher than those who survived?

Reply:
```suggestion
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 were elevated at admission in adult COVID-19 patients, who ultimately deceased, compared to those who survived[@doi:10/ggnxb3].

IL-6 levels remained higher on average, in those patients, who ultimately deceased, through the course of their hospitalization [@doi:10/ggnxb3].
```

Reply:
```suggestion
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately deceased had higher IL-6 levels at admission than those who recovered [@doi:10/ggnxb3].
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately deceased [@doi:10/ggnxb3].
```
Totally shaking things up! Does this help?

Reply:
I like this A LOT better!